app/content.tsx

```diff
@@ -0,0 +1,495 @@
+    icons: [
+      {
+        iconImage: "/usd_icon.svg",
+        iconAlt: "USDC Icon",
+        value: "9000 USDC",
+      },
+      {
+        iconImage: "/terminal_white.svg",
+        iconAlt: "Terminal White",
+        value: "Arcadia - Aerodrome Integrations",
+      },
+      {
+        iconImage: "/clinical_notes_white.svg",
+        iconAlt: "Clinical Notes White",
+        value: "zzykxx",
+      },
+    ],
+    cips: ["Swap", "Critical", "Dexes"],
+  },
+  {
+    imageSrc: "/code_hawks.png",
+    imageAlt: "Code Hawks Icon",
+    title: "CodeHawks",
+    date: "25/04/2024",
+    score: "Medium",
+    text: "Ignoring the Well Function logic for a ratio of reserves calculation",
+    icons: [
+      {
+        iconImage: "/usd_icon.svg",
+        iconAlt: "USDC Icon",
+        value: "5000 USDC",
+      },
+      {
+        iconImage: "/terminal_white.svg",
+        iconAlt: "Terminal White",
+        value: "Beans - Stalk: Dive Into Basin",
+      },
+      {
+        iconImage: "/clinical_notes_white.svg",
+        iconAlt: "Clinical Notes White",
+        value: "pontifex",
+      },
+    ],
+    cips: ["Swap", "Critical", "Dexes"],
+  },
+  {
+    imageSrc: "/code_hawk_a.png",
+    imageAlt: "Code Hawks Icon",
+    title: "CodeHawks",
+    date: "25/04/2024",
+    score: "High",
+    text: "Ignoring the Well Function logic for a ratio of reserves calculation",
+    icons: [
+      {
+        iconImage: "/usd_icon.svg",
+        iconAlt: "USDC Icon",
+        value: "3000 USDC",
+      },
+      {
+        iconImage: "/terminal_white.svg",
+        iconAlt: "Terminal White",
+        value: "Beans - Stalk: Dive Into Basin",
+      },
+      {
+        iconImage: "/clinical_notes_white.svg",
+        iconAlt: "Clinical Notes White",
+        value: "pontifex",
+      },
+    ],
+    cips: ["Swap", "Critical", "Dexes"],
+  },
+  {
+    imageSrc: "/code_hawks_b.png",
+    imageAlt: "Code Hawks Icon",
+    title: "CodeHawks",
+    date: "25/04/2024",
+    score: "Low",
+    text: "Ignoring the Well Function logic for a ratio of reserves calculation",
+    icons: [
+      {
+        iconImage: "/usd_icon.svg",
+        iconAlt: "USDC Icon",
+        value: "1500 USDC",
+      },
+      {
+        iconImage: "/terminal_white.svg",
+        iconAlt: "Terminal White",
+        value: "Beans - Stalk: Dive Into Basin",
+      },
+      {
+        iconImage: "/clinical_notes_white.svg",
+        iconAlt: "Clinical Notes White",
+        value: "pontifex",
+      },
+    ],
+    cips: ["Swap", "Critical", "Dexes"],
+  },
+];
+export const cipsContet = [
+  {
+    title: "Zwappi - Derivatives DEX",
+    imageSrc: "/terminal_black.png",
+    imageAlt: "Terminal Black Icon",
+  },
+  {
+    title: "Phoenix Shadow",
+    imageSrc: "/clinical_notes_black.png",
+    imageAlt: "Clinical Notes Black Icon",
+  },
+];
+
+export const vulnerabilitiesCard = [
```

Comment:
Can you add `suggestedChanges` here? Also, we need `certainityScore` as well since this is what we will get from the backend.

Reply:
done, pls check